features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/MessagesView.kt

```diff
@@ -247,6 +248,13 @@ fun MessagesView(
         onUserDataClick = onUserDataClick,
     )
     ReinviteDialog(state = state)
+
+    // Make sure we hide the keyboard when the view is disposed
+    DisposableEffect(Unit) {
```

Comment:
Should we not find a solution directly on the FlowNode for this? :/
Also using `SoftwareKeyboardController` doesn't work?

Reply:
I can try to find a better solution to apply at the `BaseFlowNode` level, instead of hiding the keyboard when a node is disposed, maybe it should be done when a new transition is about to start so it's more immediate.

About `SoftwareKeyboardController` I'm not sure, because the underlying UI component for the composer is an `EditText` and it might not handle focus the same way as compose components.

Reply:
Lets do this for now, and will investigate a bit more later

Reply:
I think I actually have a generic implementation ready, based on the `backstack.elements` value. It seems to work on any screen and it's immediate.

Reply:
Even better then!

Reply:
How does this react with screen we want the keyboard to open right away?

Reply:
Good question, I guess it'd try to dismiss the keyboard before the action to display it on focus happens, but I'd have to double check. Can you give me an example of one of these screens?

Reply:
Actually I think we don't have except the search on the room list, but it's not a new node

Reply:
I've tested it more, and I don't think we should use this. The keyboard not closing should only concern screen where we use a view based text field, otherwise the keyboard is closed when the field is removed from composition. 
If we try to open a keyboard quickly on a newly pushed screen, the `SoftwareKeyboardController.hide` will be triggered and so the keyboard will appear and disappear right away.